ts/app.ts

```diff
@@ -1,15 +1,17 @@
+import { ElementInterface, ListInterface, TimeOptions } from "./interfaces.js";
+
 const clear = document.querySelector(".clear");
 const dateElement = document.getElementById("date");
 const list = document.getElementById("list");
-const input = document.getElementById("input");
+const input = document.getElementById("input") as HTMLInputElement;
 
 //Classes names
 const CHECK = "fa-check-circle";
 const UNCHECK = "fa-circle-thin";
 const LINE_THROUGH = "lineThrough";
```

Comment:
All constants can be exported to a `constants.ts` and imported here, also use `SNAKE_CASE` for const

Reply:
done